src/styles/_catalog.scss

```diff
@@ -14,6 +14,13 @@
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
+    transition-duration: 0.3s;
+    transition-property: transform, color;
+
+    &:hover {
+      transform: scale(1.2);
+      color: #34568b;
```

Comment:
Create variables for colors

Reply:
ceate file variables with variables for color header and catalog